app/src/components/progress-circle/styles.ts

```diff
@@ -0,0 +1,51 @@
+import { css, keyframes } from "@emotion/react";
+
+import type { Size } from "./types";
+
+export const SIZES = {
+  S: 18,
+  M: 32,
+};
+
+export const STROKE_WIDTHS = {
+  S: 2,
+  M: 3,
+};
+
+export const CENTER = (size: Size) => SIZES[size] / 2;
+export const RADIUS = (size: Size) => SIZES[size] / 2 - STROKE_WIDTHS[size];
+export const CIRCUMFERENCE = (size: Size) => 2 * Math.PI * RADIUS(size);
+export const DASH_SHORT = (size: Size) => CIRCUMFERENCE(size) * 0.25;
+export const DASH_LONG = (size: Size) => CIRCUMFERENCE(size) * 0.75;
```

Comment:
not sure if I follow the uppercasing here. Could we refactor this to be all CSS based? Using a data-class to scope things is my preferred mechanism.